core/code/boot.js

```diff
@@ -205,6 +205,29 @@ function prepPluginsToLoad () {
   };
 }
 
+var toolboxElement;
+window.sortToolbox = function () {
+  var children = Array.prototype.slice.call(toolboxElement.children);
+  var sortedChildren = children.slice().sort(function (x, y) {
+    return x.innerText.localeCompare(y.innerText);
+  });
+  if (
+    sortedChildren.some(function (item, index) {
+      return item !== children[index];
+    })
+  ) {
+    sortedChildren.forEach(function (child) {
+      toolboxElement.removeChild(child);
+      toolboxElement.appendChild(child);
+    });
+  }
+}
+function setupToolboxSort() {
+  toolboxElement = $('#toolbox')[0];
+  window.setTimeout( ()=> window.sortToolbox(), 1000);
```

Comment:
We don't use arrow functions in core.

```suggestion
  window.setTimeout(window.sortToolbox, 1000);
```